libraries/package-extractor/src/PackageExtractor.ts

```diff
@@ -483,8 +499,13 @@ export class PackageExtractor {
           path.join(packageJsonRealFolderPath, 'package.json')
         );
 
+        const targetSubspace: IExtractorSubspace | undefined = subspaces?.find(
+          (p) => p.pnpmInstallFolder && Path.isUnder(packageJsonFolderPath, p.pnpmInstallFolder)
```

Comment:
Just curious: would it be possible to have multiple "subspaces" here?

It's related if there are any nested projects in a monorepo (which is not recommended)

Reply:
I don't think so, because each subspace's pnpmInstallFolder will be located at common/temp and they won't be nested right?

We can't have multiple subspaces in following logic, or it will have some issues 
> One dependency from one subspace might be resolved to another subspace, which is not expected